src/ansible_navigator/configuration_subsystem/definitions.py

```diff
@@ -134,16 +138,21 @@ class SubCommand(SimpleNamespace):
     epilog: Union[None, str] = None
 
 
-class ApplicationConfiguration(SimpleNamespace):
+@dataclass
+class ApplicationConfiguration:
+    # pylint: disable=too-many-instance-attributes
```

Comment:
Can we globally kill this? It's pretty silly. Especially for dataclasses. They store things... data, even!

Reply:
I'm game, although it might bring attention to the need for a class that should have some subclasses, I don't think I'd ever pick up on it.